### PR TITLE
Extract last 2 plugins and clean up others

### PR DESCRIPTION
**Pull request by Ailrun**
Extract
- ormolu plugin
- module name plugin

(last 2 plugins reside in `default` directory) and clean up cabal files & tests of other plugins.

Although I think it's generally better to use `.hs` extensions even for golden files so that we can have syntax highlight and others, I didn't touch Wingman's tests as I am not sure whether any of @isovector's tools depend on golden test extensions or not. Is it OK to update Wingman's tests so that we have `.expected.hs` or similar.

### CODE REVIEW

File: hls-test-utils/src/Test/Hls.hs [hunk 1]
```diff
@@ -67,6 +72,24 @@ gitDiff fRef fNew = ["git", "-c", "core.fileMode=false", "diff", "--no-index", "
...
+  -> TestName
+  -> FilePath
+  -> FilePath
+  -> FilePath
+  -> FilePath
+  -> (TextDocumentIdentifier -> Session ())
+  -> TestTree
+goldenWithHaskellDoc plugin title testDataDir path desc ext act =
+  goldenGitDiff title (testDataDir </> path <.> desc <.> ext)
+  $ runSessionWithServer plugin testDataDir
```
Comment (Ailrun) [inline hls-test-utils/src/Test/Hls.hs:86]: there should be a version using `runSessionWithServerFormatter`.

**Review by jneira — APPROVED** [review]

**Comment by isovector** [comment]
@Ailrun completely fine to rename the Wingman tests! thanks for tackling this!

File: plugins/hls-tactics-plugin/test/Utils.hs [hunk 2]
```diff
@@ -61,7 +61,7 @@ codeActionTitle (InR(CodeAction title _ _ _ _ _ _ _)) = Just title
 mkTest
     :: Foldable t
     => String  -- ^ The test name
-    -> FilePath  -- ^ The file to load
+    -> FilePath  -- ^ The file basename to load
```
Comment (Ailrun) [inline plugins/hls-tactics-plugin/test/Utils.hs:64]: It's not "basename", it's a stem.

File: plugins/hls-eval-plugin/test/testdata/hie.yaml [hunk 3]
```diff
@@ -1,4 +1,3 @@
 cradle:
-  cabal:
-    - path: "././"
-      component: "lib:test"
+  direct:
+    arguments: []
```
Comment (Ailrun) [inline plugins/hls-eval-plugin/test/testdata/hie.yaml:3]: Should list `Util.hs`

**Pull request merged**